client/modules/components/src/Wikitext.svelte

```diff
@@ -47,10 +47,17 @@
   /** Shows render performance information if true. */
   export let debug = false
 
+  /**
+   * If true, the wikitext will be inserted into a shadow DOM so that it is
+   * entirely isolated from the outside DOM from which it is mounted.
+   */
+  export let isolated = false
```

Comment:
Why is this false by default?

Reply:
It's a generic component, it's just only being used for the live preview right now. Currently, styling a page or something breaks out of that page's container, so this is just following default behavior. We _could_ actually change that, but that's probably another issue because that has a fair amount of ramifications (e.g. that always requires JS to do, breaks a bunch of articles, etc.).